hono/src/utils/swaggerSchema.ts

```diff
@@ -0,0 +1,300 @@
+  method: 'delete',
+  path: '/items',
+  description: 'delete all items',
+  summary: 'delete items',
+  security: [{ [bearerAuth.name]: [] }],
+  request: {
+    params: z.object({ id: ItemIdSchemaDel }),
+  },
+  responses: {
+    200: {
+      description: 'Object with user data.',
+      content: {
+        'application/json': {
+          schema: ItemSchema,
+        },
+      },
+    },
+    204: {
+      description: 'No content - successful operation',
+    },
+  },
+})
+
+registry.registerPath({
+  method: 'get',
+  path: '/outfits',
+  description: 'return all outfits',
+  summary: 'get outfits',
+  security: [{ [bearerAuth.name]: [] }],
+  responses: {
+    200: {
+      description: 'Object with user data.',
+      content: {
+        'application/json': {
+          schema: OutfitSchema,
+        },
+      },
+    },
+    204: {
+      description: 'No content - successful operation',
+    },
+  },
+})
+
+registry.registerPath({
+  method: 'post',
+  path: '/outfits',
+  description: 'post all outfits',
+  summary: 'get items',
+  security: [{ [bearerAuth.name]: [] }],
+  /*
+  request: {
+    params: z.object({ id: UserIdSchema }),
+  },
+  */
+  responses: {
+    200: {
+      description: 'Object with user data.',
+      content: {
+        'application/json': {
+          schema: OutfitSchema,
+        },
+      },
+    },
+    204: {
+      description: 'No content - successful operation',
+    },
+  },
+})
+
+registry.registerPath({
+  method: 'put',
+  path: '/outfits',
+  description: 'put all items',
+  summary: 'put items',
+  security: [{ [bearerAuth.name]: [] }],
+  request: {
+    params: z.object({ id: OutfitIdSchema }),
+  },
+  responses: {
+    200: {
+      description: 'Object with user data.',
+      content: {
+        'application/json': {
+          schema: OutfitSchema,
+        },
+      },
+    },
+    204: {
+      description: 'No content - successful operation',
+    },
+  },
+})
+
+registry.registerPath({
+  method: 'delete',
+  path: '/outfits',
+  description: 'delete all items',
+  summary: 'delete items',
+  security: [{ [bearerAuth.name]: [] }],
+  request: {
+    params: z.object({ id: OutfitIdSchemaDel }),
+  },
+  responses: {
+    200: {
+      description: 'Object with user data.',
+      content: {
+        'application/json': {
+          schema: OutfitSchema,
+        },
+      },
+    },
+    204: {
+      description: 'No content - successful operation',
+    },
+  },
+})
+
+async function getOpenApiDocumentation() {
+  const generator = new OpenApiGeneratorV3(registry.definitions)
+
+  return await generator.generateDocument({
+    openapi: '3.0.0',
+    info: {
+      version: '1.0.0',
+      title: 'My API',
+      description: 'This is the API',
+    },
+    servers: [{ url: 'http://127.0.0.1:8787/api/v1' }],
+  })
+}
+
+export default async function writeDocumentation() {
+  const docs = await getOpenApiDocumentation()
+  const fileContent = JSON.stringify(docs)
+  try {
```

Comment:
If possible, you may be able to remove this try/catch block. The default node error handler should be able to handle this itself, and it is fine if execution is stopped if `fs` throws.